src/frontend/Ast_to_Mir.ml

```diff
@@ -3,8 +3,10 @@ open Middle
 
 (* XXX fix exn *)
 let unwrap_return_exn = function
-  | Some (ReturnType ut) -> ut
-  | x -> raise_s [%message "Unexpected return type " (x : returntype option)]
+  | Some (Middle.ReturnType ut) -> ut
+  | x ->
+      raise_s
+        [%message "Unexpected return type " (x : Middle.returntype option)]
```

Comment:
We opened Middle at the top - we shouldn't need to change lines like these, probably an artifact from modularizing the Mir type.